ERCS/erc-7590.md

```diff
@@ -0,0 +1,402 @@
+---
+eip: 7590
+title: ERC-7590 Linagee Name Registrar and Ethereum Content Delivery Network
```

Comment:
```suggestion
title: Linagee Name Registrar and Ethereum Content Delivery Network
```
Don't include the ERC# in the description

Reply:
ERCs are not the place for product promotion. Please consider if this is an appropriate to be an ERC.

Reply:
I understand. This is not meant to be promotion. I am just trying to convey our standard. I can reword to have the word "Linagee" appear less often if thats what you think would be the best course of action.

Reply:
A standard for a specific protocol doesn't need to be an ERC.  This can be specified in the protocol documentation.

Reply:
I believe this may. Its important developers of eip 4804 know how to handle permanent domains, similarly how ENS has an eip to do the same.